
I guess non-pointer FILE is not portable at least it breaks with dietlibc ...

  - Rene Rebe <rene@exactcode.de>

--- coreutils-5.2.1/src/paste.c.vanilla	2005-06-15 23:48:26.000000000 +0200
+++ coreutils-5.2.1/src/paste.c	2005-06-15 23:48:49.000000000 +0200
@@ -52,11 +52,11 @@
 /* Indicates that no delimiter should be added in the current position. */
 #define EMPTY_DELIM '\0'
 
-static FILE dummy_closed;
+static FILE* dummy_closed;
 /* Element marking a file that has reached EOF and been closed. */
-#define CLOSED (&dummy_closed)
+#define CLOSED (dummy_closed)
 
-static FILE dummy_endlist;
+static FILE* dummy_endlist;
 /* Element marking end of list of open files. */
 #define ENDLIST (&dummy_endlist)
 
